packages/story-editor/src/components/canvas/editElement.js

```diff
@@ -17,80 +17,62 @@
 /**
  * External dependencies
  */
+import { useState, forwardRef } from '@googleforcreators/react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { useUnits } from '@googleforcreators/units';
 
 /**
  * Internal dependencies
  */
-import { useState, useRef } from '@googleforcreators/react';
 import { getDefinitionForType } from '../../elements';
 import {
   elementWithPosition,
   elementWithSize,
   elementWithRotation,
 } from '../../elements/shared';
-import SingleSelectionMoveable from './singleSelectionMoveable';
 
 const Wrapper = styled.div`
   ${elementWithPosition}
   ${elementWithSize}
-	${elementWithRotation}
-	pointer-events: initial;
+  ${elementWithRotation}
+  pointer-events: initial;
```

Comment:
Not sure what this is about?

Reply:
It's just tabs vs spaces holy war 😉 

![image](https://user-images.githubusercontent.com/21036224/151953417-5697db63-143d-43d3-8af7-42271526e007.png)